sources/@roots/bud-preset-wordpress/package.json

```diff
@@ -77,13 +77,5 @@
     "@roots/bud-wordpress-manifests": "workspace:sources/@roots/bud-wordpress-manifests",
     "@wordpress/browserslist-config": "4.1.0",
     "tslib": "2.3.1"
-  },
```

Comment:
I'm starting to wonder if `peerDependenciesMeta.[package].optional` does the opposite of what i want. removing this fixes resolution issues I was having with this package in dev.